collie/cross_validation.py

```diff
@@ -145,14 +146,17 @@ def stratified_split(interactions: BaseInteractions,
                      force_split: bool = False) -> Tuple[BaseInteractions, ...]:
     """
     Split an ``Interactions`` instance into train, validate, and test datasets in a stratified
-    manner such that each user appears at least once in each of the datasets.
+    manner such that each user or item, depending on ``interactions.negative_sample_type`` (user
+    when ``negative_sample_type == 'item'``, item when ``negative_sample_type == 'user'``) appears
+    at least once in each of the datasets.
```

Comment:
Can we add this language to: 

* `ApproximateNegativeSamplingInteractionsDataLoader` L191 
* `Interactions` L262 
* `BasePipeline.calculate_loss` L591